tests/storage/hotplug.go

```diff
@@ -1341,13 +1385,33 @@ var _ = SIGDescribe("Hotplug", func() {
 					}
 				}
 			}
+			By("Checking hotplug attachment pod ratios")
```

Comment:
Do we want to verify the proper values are set for the other containers (containerdisk, etc)?  Don't have to do anything as fancy as this

Reply:
This test actually checks both container disks AND hotplug attachment containers (Note I changed the VM boot source from datavolume to container disk). If I can find a side car/virtiofs test I could modify them to check for the values.